src/Courses/CoursePage.tsx

```diff
@@ -1,22 +1,51 @@
 import React, { useEffect, useState } from 'react';
 import CourseList from './CourseList';
+import AxiosRequest from '../Hooks/AxiosRequest';
+import { map } from 'lodash';
+import { CourseObject } from './CourseInterfaces';
+import { Container, Row, Button, Col } from 'react-bootstrap';
+import { BsPlusSquare } from 'react-icons/bs';
+import Cookies from 'js-cookie';
 
 interface CoursePageProps {
 
 }
 
 export const CoursePage: React.FC<CoursePageProps> = () => {
-    const [courses, setCourses] = useState<Array<String>>([]);
+    const [courses, setCourses] = useState<Array<CourseObject>>([]);
+    const userType = Cookies.get('userType');
 
     // Get the list of courses to render.
     useEffect(() => {
-        setCourses(['Course 1', 'Course 2', 'Course 3']);
+        (async () => {
+            try {
+                let res = await AxiosRequest.get('/courses');
+                console.log(res.data.data);
+                const courses: Array<CourseObject> = map(res.data?.data, obj => new CourseObject(obj));
+
+                setCourses(courses);
+            } catch (e) {
+                console.log(e.response);
```

Comment:
should we have `console.log`
should we use a client side logger?

Reply:
I think for now, it's okay. I don't think I've used any client-side loggers, but I'll look into `loglevel` and anything you might think up.